src/converters.h

```diff
@@ -0,0 +1,47 @@
+#include <v8.h>
+#include "nan.h"
+
+using namespace v8;
+
+Local<Object> toLocalObj(Local<Value> val) {
```

Comment:
1) As it's C++ code the file should have `.hpp` extension.
2) We could hide these helper functions behind some kind of namespace.
```cpp
namespace nodezk {
  Local<Object> toLocalObj(Local<Value> val) {
    // ...
  }
}
```